Desktop version

shinyenyi opened the pull request:
-Implement html code and css styling for desktop version of project as per project requirements

Miguel-Shinyenyi committed:
Desktop version
-Implement html code and css styling for desktop version of project as per project requirements

emmiiorji commented on styles.css:
```diff
@@ -650,3 +654,278 @@ footer {
+    margin-top: 24px;
+    margin-bottom: 24px;
+    border: 1px solid #212121;
+  }
```
In order to follow the design a lot more closely, kindly modify the width and color of the recent-works horizontal rule to match the design. Take note of the alignments.

Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

So, let's follow the original vision if it's pretty and works, it will be a wonderful website.


Your design:
![image](https://user-images.githubusercontent.com/20554583/187028286-bc5d50d1-095f-448b-aca6-3016f9898801.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187028412-d64c0de0-4114-4330-b201-a5545fbd6bfe.png)

emmiiorji commented on index.html:
```diff
@@ -90,7 +99,7 @@ <h1 class="works-project-title">Profesional Art Printing Data</h1>
         <button class="works-project-button">See Project</button>
       </li>
```
To match the design on Figma, kindly adjust the alignment between the list of languages and also the `see project` button's margin from them. Check the comment above for more reason you should consider sticking to a given design as much as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187028662-6c1e0799-b2a3-478e-a408-63f72c4c7cae.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187028680-c50d545b-02c2-4ebd-9c60-709c34126c8a.png)

emmiiorji commented on index.html:
```diff
@@ -134,6 +143,36 @@ <h1 class="works-project-title">Profesional Art Printing Data</h1>
         </div>
         <button class="works-project-button">See Project</button>
       </li>
```
You've done great implementing some of the button interactions. However, the color are yet to exactly match the design. Kindly modify it to do so.

Your button-pressed interaction:
![image](https://user-images.githubusercontent.com/20554583/187028859-63429542-2e09-44cb-aadb-46d72ef13e21.png)

Your button-enabled state:
![image](https://user-images.githubusercontent.com/20554583/187028954-4c6ab5e3-d0c1-4140-b587-d76ad51e5bee.png)


Figma interactions:
![image](https://user-images.githubusercontent.com/20554583/187028865-2f190cb0-1401-44f4-8457-c88dfd8104e9.png)

emmiiorji commented on index.html:
```diff
@@ -134,6 +143,36 @@ <h1 class="works-project-title">Profesional Art Printing Data</h1>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button class="works-project-button">See Project</button>
+      </li>
     </ul>
   </section>
```
Kindly modify the project section to match that of the design. You can observe the margins and also observe that the project images are portrait-like and not landscape.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187029268-9a15419c-b21d-428a-9a65-9804d75a5caa.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029292-26597039-d842-4c15-b924-79af54840527.png)

emmiiorji commented on index.html:
```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
     <button class="about-me-button">Get My Resume</button>
     <div class="about-me-line"></div>
-    <ul class="about-me-lists">
-      <li class="about-me-list-header">
```
Please, also adjust the about section to match that of your chosen design template. Kindly check the comment above for more reason you should consider sticking closely to a given design.

Your design: 
![image](https://user-images.githubusercontent.com/20554583/187029541-328ea441-704d-44ae-8e39-5fb351654126.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029524-d7f71ce8-b344-4707-8eef-743518d5eb55.png)

emmiiorji commented on index.html:
```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
+      <ul class="about-me-lists">
+        <li class="about-me-list-header">
+          <div class="star"></div>
+          <h1>Language</h1>
+        </li>
+        <li class="about-me-list-item">Javascript</li>
+        <li class="about-me-list-item">Ruby</li>
+        <li class="about-me-list-item">Html</li>
+        <li class="about-me-list-item">Css</li>
```
Please, modify your skills section to also match the design. The alignment of the design looks better. Also adjust the font sizes to match Figma.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187029719-92d51146-ddc4-453e-9c00-cefd04b44499.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029737-580d78e6-4fbb-48c7-83be-7639cb5a456a.png)

emmiiorji commented on index.html:
```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
-      <input class="contact-form-email" type="email" name="email" placeholder="Email address" required>
-      <textarea class="contact-form-message" rows="3" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-      <button class="contact-form-button" type="submit">Get in touch</button>
+      <div class="contact-form-inputs">
+        <input class="contact-form-name" type="text" name="name" placeholder="Full name" 
+        maxlength="30" required>
+        <input class="contact-form-email" type="email" name="email" 
+          placeholder="Email address" required>
+        <textarea class="contact-form-message" rows="3" name="message" 
+          placeholder="Enter text here" maxlength="500" required></textarea>
```
Considering the contact section, the template's design looks better. Kindly stick to it. 

Kindly take note of:

- The background color of the form fields
- The alignment of the text `i'm always interested...`
- The fonts

Your design:
![image](https://user-images.githubusercontent.com/20554583/187029881-5bdb6b21-ae8b-41db-98ae-4e94cf1e9b15.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029873-8b1ca1c4-d6f9-4642-950c-ad3b595ea900.png)

Miguel-Shinyenyi committed:
Add css animation and transition
-add transition to the headline page title, job and paragraph
-add scale up to work projects
-add rotate to the star

emmiiorji commented on index.html:
```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
+        <button class="contact-form-button" type="submit">Get in touch</button>
+      </div>
     </form>
   </section>
   <footer>
```
Please, also modify the color of the horizontal rule at the footer to match your chosen Figma template. Check my comment above for more reasons you should consider staying close to given design.#

Your design:
![image](https://user-images.githubusercontent.com/20554583/187030072-66dd9bf4-3a5f-492a-bbdb-e24c77daf29c.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187030059-17ac5715-5402-4be5-8ab0-f32841f1a6b0.png)

emmiiorji reviewed (changes requested):
Hi @shinyenyi, 
# Status: Changes Required ♻️

Great progress so far!
However, there are a few issues that you still need to work on to make the project stand out but you are almost there!

![a little more](https://user-images.githubusercontent.com/20554583/187027333-03214843-4889-4d62-be02-bf1b7cab5ce0.gif)


## Good Highlights ✅

- Descriptive PR
- Linter checks are passing
- Good commit messages


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

You can also consider the following:
- I suggest that fill out all sections of your README file or remove the ones that haven't been filled out yet. This will make it more concise and straight to the point
![image](https://user-images.githubusercontent.com/20554583/187027669-78a15f2c-ab50-4e67-ab73-4ac1eb8774c4.png)

-  I recommend that you include the MIT license file in the root folder of your project so that the link given in the License section of your README file wouldn't be a broken one.

You can customize [this](https://github.com/microverseinc/readme-template/blob/master/MIT.md). You can also set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).
![image](https://user-images.githubusercontent.com/20554583/187027769-38f071af-c385-4bcc-900d-d2115193f718.png)


- Considering that the footnote in the README file is a note to the author, you may want to remove it since it might not be serving the visitors of your repo a purpose as regards the project.
![image](https://user-images.githubusercontent.com/20554583/187027846-cbc3fd48-60b6-45cc-ab67-d6b5492f45c6.png)

_Every optional comment is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Miguel-Shinyenyi committed:
fix linter errors

shinyenyi committed:
Merge pull request #6 from shinyenyi/added_animation-transition

Add css animation and transition

Miguel-Shinyenyi committed:
make requested changes
-Remove unused details from readme file
-modify width and color of recent works line
-modify alignment in multi-post-storie button and list of languages
-modify button color when pressed or hover
-modify works section padding and its cards heights as requested
-modify about me padding
-modify about me technologies alignment
-modify contact form background color, alignment and fonts
-modify footers horizontal rule

danmainah commented on index.html:
```diff
@@ -45,30 +55,29 @@ <h1 class="greetings">Hey There. I’m Miguel</h1>
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
+            <ul class="languages">
```
- [ ] [optional] Kindly check the design of your mobile version, it broken check the comparison below:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/187034728-cafaaa30-c111-48cd-a6a8-f9847c1d2872.png) |  ![image](https://user-images.githubusercontent.com/61408860/187034760-f5679c4a-d4b4-43e0-b181-6dd5dd7f5473.png)

danmainah reviewed (approved):
Hi @shinyenyi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
